packages/example/src/pages/index.tsx

```diff
@@ -4,10 +4,12 @@ import {
   withDefault,
   BooleanParam
 } from 'next-query-params';
+import {useRouter} from 'next/dist/client/router';
```

Comment:
Let's use the suggested way to import the router:

```suggestion
import {useRouter} from 'next/router';
```